src/shortcuts.js

```diff
@@ -360,7 +360,7 @@ export const globalShortcuts = perma(() => [
   {
     id: 'cursorDown',
     name: 'Cursor Down',
-    keyboard: 'ArrowDown',
+    keyboard: { key: 'ArrowDown', meta: true },
```

Comment:
These should ideally be in their own PR. Something may have happened with how we ordered the commits. Not a big deal though, so just leave for now.